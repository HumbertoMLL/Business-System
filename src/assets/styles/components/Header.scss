@import '../Vars.scss';

.header {
  display: grid;
  width: 100%;
  height: 100%;
  grid-template-columns: 1fr 1fr;
  background-color: $color-suave;

  &__logo {
      grid-column: 1;
      margin-top: .5em;
      margin-bottom: .5em;
      margin-left: 1.5em;
  & img {
      width: 5em;
      height: auto;
  }
  }
  &__user {
      grid-column: 2;
      display: flex;
      flex-direction: row;
      align-items: center;
      justify-content: flex-end;
      margin-right: 2em;
  & img {
      width: 4em;
      height: auto;
  }
  & p {
      font-family: $fuente;
      font-size: 24px;
      font-weight: 600;
  }
  }
}