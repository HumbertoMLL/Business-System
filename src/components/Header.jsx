import React from 'react';
import '../assets/styles/App.scss';
import '../assets/styles/components/Header.scss';
// import logo from '../static/github.svg';
import userIcon from '../static/woman.svg';

const Header = () => (

  <header className='header'>
    <div className='header__logo'>
      <img src='https://firebasestorage.googleapis.com/v0/b/humbertostark21.appspot.com/o/github.svg?alt=media&token=a96f56f0-88d9-4038-a583-c77dd231e787' alt='' />
    </div>
    <div className='header__user'>
      <img src={userIcon} alt='' />
      <p>Username</p>
    </div>
  </header>

);

export default Header;
