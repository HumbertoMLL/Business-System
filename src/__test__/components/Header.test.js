import React from 'react';
import { mount } from 'enzyme';
import { create } from 'react-test-renderer';
import Header from "../../components/Header";

describe('<Header />', () => {
  test('Render del componente Header', () => {
    const header = mount(<Header />);
    expect(header.length).toEqual(1);
  });
});

describe('Header Snapshot', () => {
  test('Comprobar la UI del componente Header', () => {
    const header = create(<Header />);
    expect(header.toJSON()).toMatchSnapshot();
    //Para actualizar el snapshot cuando actualizamos el html
    //Usamos el comando en la terminal
    //jest --updateSnapshot
  });
});
