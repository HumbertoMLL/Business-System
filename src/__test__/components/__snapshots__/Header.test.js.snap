// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Snapshot Comprobar la UI del componente Header 1`] = `
<header
  className="header"
>
  <div
    className="header__logo"
  >
    <img
      alt=""
      src={Object {}}
    />
  </div>
  <div
    className="header__user"
  >
    <img
      alt=""
      src={Object {}}
    />
    <p>
      Username
    </p>
  </div>
</header>
`;
